vulyk/blueprints/gamification/__init__.py

```diff
@@ -178,14 +178,34 @@ def unseen_events() -> flask.Response:
     if isinstance(user, User):
         return utils.json_response({
             'events': map(
-                lambda e: e.to_dict(),
-                EventModel.get_unread_events(flask.g.user))})
+                lambda e: utils.blacklist(e.to_dict(), ["answer"]),
```

Comment:
Why not creating another method `Event.to_presentation_dict()` or something like that?
I mean I don't see the raison d'être of `utils..blacklist()`.